src/main/java/net/ccbluex/liquidbounce/injection/mixins/minecraft/entity/MixinEntity.java

```diff
@@ -71,6 +74,11 @@ public static Vec3d movementInputToVelocity(Vec3d movementInput, float speed, fl
     @Shadow
     public abstract double getZ();
 
+    @Redirect(method = "bypassesLandingEffects", at = @At(value = "INVOKE", target = "Lnet/minecraft/entity/Entity;isSneaking()Z"))
```

Comment:
Do not use `@Redirect`. It cannot be stacked and might cause mod incompatibility. Use this instead: https://github.com/LlamaLad7/MixinExtras/wiki/ModifyExpressionValue